vaadin-testbench-unit/src/main/kotlin/com/vaadin/testbench/unit/internal/PrettyPrintTree.kt

```diff
@@ -150,6 +151,17 @@ fun Component.toPrettyString(): String {
         val outerHtml: String = this.element.outerHTML.trim().replace(Regex("\\s+"), " ")
         list.add(outerHtml.ellipsize(100))
     }
+    if (this is InternalServerError) {
+        println(this.element.outerHTML)
+        val view = this.element.getChild(0).outerHTML.replace(Regex("(?s).* trying to navigate to '([^']+).*"), "$1")
+        val errorMessage = this.element.getChild(0).outerHTML.
+            replace(Regex("(?s).* with the (?:root cause|exception message) '([^']+).*"), "$1")
+        val stackTrace: String = this.element.outerHTML.trim().substringAfter("<pre>").substringBefore("</pre>")
+        //list.add(outerHtml.ellipsize(100))
+        list.add("targetView='$view'")
+        list.add("error='$errorMessage'")
+        list.add("stacktrace='${stackTrace.substringAfter("\n")}'")
+    }
```

Comment:
Perhaps we should just create test error page extending InternalServerError that would store the data into parameters.
Then they would be added automatically.